src/js/rebuses.js

```diff
@@ -902,6 +902,11 @@ const rebuses = [
     symbols: ['🦀', '+', '🎂'],
     words: ['crab', 'cake'],
     hint: 'A delicious appetizer'
+  },
+  {
+    symbols: ['⭐', '+', '⚔','s'],
```

Comment:
Pretty sure your build failure will be resolved if you put a space between `,` and `'s'`.

Reply:
Oh, ok, thanks

Reply:
Looks like that worked. Way to go! Now you just have to wait for somebody with write access to merge it. 😄

Reply:
> 
> 
> Looks like that worked. Way to go! Now you just have to wait for somebody with write access to merge it. 😄

Thanks samuei ! 👍